db/migrate/20201104002858_remove_status_from_orders.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveStatusFromOrders < ActiveRecord::Migration[5.2]
```

Comment:
Sometimes I think deleting a migration and then recreating it with the type you want is much easier than creating a migration that changes a column 👍

Reply:
I spent... Longer than I care to admit trying to change the type to `integer` before I thought to delete the column lol.

Reply:
Thanks, @KieraAllen! I didn't think of that, and have only changed it rather than try this.